projects/batfish/src/main/java/org/batfish/grammar/cisco/CiscoControlPlaneExtractor.java

```diff
@@ -4116,14 +4340,36 @@ public void exitRedistribute_ospf_bgp_tail(Redistribute_ospf_bgp_tailContext ctx
   }
 
   @Override
+  public void exitRedistribute_rip_bgp_tail(Redistribute_rip_bgp_tailContext ctx) {
+    BgpProcess proc = currentVrf().getBgpProcess();
+    if (_currentPeerGroup == proc.getMasterBgpPeerGroup()) {
```

Comment:
`==` or `.equals`?

Reply:
==. This is meant to be compare-by-reference. I will add comment.